requirements.txt

```diff
@@ -63,7 +63,7 @@ stable_baselines3
 colorama==0.4.3
 urllib3==1.25.11
 
-git+https://github.com/rlworkgroup/metaworld.git@master#egg=metaworld
+git+https://github.com/rlworkgroup/metaworld.git@d9a75c451a15b0ba39d8b7a8b6d18d883b8655d8#egg=metaworld
```

Comment:
It is not an option to fix a pip version for them instead of a SHA?

Reply:
No, unfortunately they don't have pip releases yet.